### PR TITLE
Tickets/dm 37171

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -16,14 +16,14 @@ repos:
...
-    rev: 5.10.1
+    rev: 5.12.0
     hooks:
       - id: isort
         name: isort (python)
         args: [--settings-path=pyproject.toml]
 
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.991
+    rev: v1.0.0
```
Comment (r-owen) [inline .pre-commit-config.yaml:26]: Are we upgrading black and mypy yet?

File: python/lsst/ts/mtdome/enums.py [hunk 2]
```diff
@@ -42,14 +42,12 @@ class InternalMotionState(enum.IntEnum):
...
     """
 
-    DISABLED = enum.auto()
     STATIONARY = enum.auto()
 
 
 # Dict holding translations from motion states, that the lower level
 # controllers can have, to MotionState.
 motion_state_translations = {
-    "DISABLED": MotionState.ERROR,
```
Comment (r-owen) [inline python/lsst/ts/mtdome/enums.py:52]: Nice to see another of these go. Any chance of eliminating the last one?

**Review by r-owen — APPROVED** [review]
Looks fine. Just wondering if it's safe to update black and mypy yet (I should have asked that in a previous review I did for you).

File: .pre-commit-config.yaml (hunk 1 above)
Comment (wvreeven) [reply .pre-commit-config.yaml:26]: It seems I am a little too pro-active. We will discuss this at standup tomorrow.

File: python/lsst/ts/mtdome/enums.py (hunk 2 above)
Comment (wvreeven) [reply python/lsst/ts/mtdome/enums.py:52]: I'm afraid not. The vendor doesn't want to rename their states.

**Pull request merged**